src/wp-includes/rest-api.php

```diff
@@ -1203,6 +1203,26 @@ function rest_parse_hex_color( $color ) {
 	return $color;
 }
 
+/**
+ * Sanitize email.
+ *
+ * @since 5.7.0
+ *
+ * @param string $email Email address
+ * @return string Valid email address.
+ */
+function rest_sanitize_email( $email ) {
+
+	if ( false !== is_email( $email ) ) {
```

Comment:
`is_email()` requires a `string` and is using `strlen()` which requires a non-nullable `string`. To avoid PHP 8.1 deprecation notices and later PHP 9 fatal error, I'd suggest first checking if the given value is a string:

```suggestion
	if ( ! is_string( $email ) || false !== is_email( $email ) ) {
```